ramsete/robotcontainer.py

```diff
@@ -4,66 +4,74 @@
-
-from wpimath.trajectory.constraint import DifferentialDriveVoltageConstraint
-from wpimath.trajectory import TrajectoryConfig, TrajectoryGenerator, Trajectory
-
 from wpimath.geometry import Pose2d, Rotation2d, Translation2d
+from wpimath.trajectory.constraint import DifferentialDriveVoltageConstraint
+from wpimath.trajectory import TrajectoryConfig, TrajectoryGenerator
 
-from subsystems.drivetrain import Drivetrain
+from commands2 import InstantCommand, RunCommand, RamseteCommand, cmd
+from commands2.button import JoystickButton
 
+from subsystems.driveSubsystem import DriveSubsystem
 import constants
 
 
 class RobotContainer:
-
     """
-    This class hosts the bulk of the robot's functions. Little robot logic needs to be
-    handled here or in the robot periodic methods, as this is a command-based system.
-    The structure (commands, subsystems, and button mappings) should be done here.
+    This class is where the bulk of the robot should be declared. Since Command-based is a
+    "declarative" paradigm, very little robot logic should actually be handled in the {@link Robot}
+    periodic methods (other than the scheduler calls). Instead, the structure of the robot (including
+    subsystems, commands, and button mappings) should be declared here.
     """
 
     def __init__(self):
-        # Create the driver's controller.
-        self.driverController = XboxController(constants.kDriverControllerPort)
+        # The robot's subsystems
+        self.robotDrive = DriveSubsystem()
 
-        # Create an instance of the drivetrain subsystem.
-        self.robotDrive = Drivetrain()
+        # The driver's controller.
+        self.driverController = XboxController(constants.kDriverControllerPort)
 
-        # Configure and set the button bindings for the driver's controller.
+        # Configure the button bindings
         self.configureButtons()
 
-        # Set the default command for the drive subsystem. It's default command will allow
-        # the robot to drive with the controller.
-
+        # Configure default commands
+        # Set the default drive command to split-stick arcade drive
         self.robotDrive.setDefaultCommand(
+            # A split-stick arcade command, with forward/backward controlled by the left
+            # hand, and turning controlled by the right.
             RunCommand(
                 lambda: self.robotDrive.arcadeDrive(
-                    -self.driverController.getRawAxis(1),
-                    self.driverController.getRawAxis(2) * 0.65,
+                    -self.driverController.getLeftY(),
+                    -self.driverController.getRightX(),
                 ),
                 self.robotDrive,
             )
         )
 
+    def configureButtons(self):
+        """
+        Use this method to define your button->command mappings. Buttons can be created by
+        instantiating a {@link edu.wpi.first.wpilibj.GenericHID} or one of its subclasses ({@link
```

Comment:
Java links

Reply:
Ugh git... I fixed that once but then did some git jiggerypokery and I guess it got reverted.  Fixed!